test/jdk/java/lang/Object/FinalizationOption.java

```diff
@@ -89,7 +89,7 @@ static boolean checkFinalizerCalled(boolean expected) {
         for (int i = 0; i < 100; i++) {
```

Comment:
Hello Tom, I think this entire loop can be replaced by using the `jdk.test.lib.util.ForceGC` utility class available in the tests. That class has the necessary knowledge of using the jtreg timeout factor. I think you could replace this loop with something like (I haven't tried it):
```java

 ForceGC.wait(() -> finalizerWasCalled);
```

Reply:
Thanks for the tip.  I'll take a look at.  Even if there isn't something that's exactly right I think having a general waitForReferenceProcessing helper would be a good idea for any tests which do this.